mmv1/third_party/terraform/services/gkehub2/data_source_google_gke_hub_membership.go

```diff
@@ -0,0 +1,39 @@
+package gkehub2
```

Comment:
`resourceGKEHub2MembershipRead` is child to the gkehub package not the gkehub2 package

https://github.com/hashicorp/terraform-provider-google-beta/blob/7a422d4064271979378ed3d149031c292f3253d2/google-beta/services/gkehub/resource_gke_hub_membership.go#L294